07-writing-technical-content.html.md

```diff
@@ -47,24 +47,69 @@ Be as clear as possible. Use simple words and phrases, avoid gerunds and hard-to
+* Use sentence case for all headings other than H1 titles. Avoid the use of gerunds when starting headings where possible. As previously stated, you should be addressing the user. "Creating a gateway" is ambiguous. "Create a gateway" is an action, an instruction, and is addressed to the user.
 * Refrain from the use of symbols (code markers, carets, brackets) in headings, and where possible, refrain from the use of any punctuation markings at all.
 
 #### Ordered Lists
 
-Only use ordered lists for step-by-step instructions. Separate steps into logical chunks, with no more than two related actions per step. When additional explanation or a screenshot is necessary, use a line break inside the list item. The line items should each follow on the subsequent line, and not have extraneous newlines between them. Ordered lists, in Markdown, should use the 1. 1. 1. formatting so that the numbers are generated on build.
+Only use ordered lists for step-by-step instructions. Separate steps into logical chunks, with no more than two related actions per step. When additional explanation or a screenshot is necessary, use a line break inside the list item. The line items should each follow on the subsequent line, and not have extraneous new lines between them. Ordered lists, in Markdown, should use the 1. 1. 1. formatting so that the numbers are generated on build.
 
 #### Unordered Lists
 
-Use unordered lists to display examples or multiple notes. If an unordered list comprises more than 10 items, use a table instead. The line items should each follow on the subsequent line, and not have extraneous newlines between them.
+Use unordered lists to display examples or multiple notes. If an unordered list comprises more than 10 items, use a table instead. The line items should each follow on the subsequent line, and not have extraneous new lines between them.
+
+#### Cross References
+
+When cross-referencing another page from strongDM documentation, link to the page and use the actual page title or section header name in the link. Links are not underlined or italicized.
+
+- Please read the [Connection to SQL Datasources](https://www.strongdm.com/docs/user-guide/connect-to-resources/connection-sql-datasources) guide for specific SQL connection requirements.
+- See section **Database Overrides**.
+
+When cross-referencing a non-strongDM page, such as documentation or a technical spec from another company, link to the page and be as descriptive as possible in and around the link so that the reader knows they will be directed away from strongDM. Use the other company’s name and the page title or section.
+
+- You can read more about [AWS KMS](https://docs.aws.amazon.com/kms/latest/cryptographic-details/intro.html) in Amazon’s documentation.
+
+#### Images
+Images should be saved as PNG. There is no default size, but make sure to run images through tinypng.com before uploading them, so that your images are not gigantic.
```

Comment:
```suggestion

Images should be saved as PNG. There is no default size, but make sure to run images through tinypng.com before uploading them, so that your images are not gigantic.
```